src/test/java/com/aws/greengrass/clientdevices/auth/iot/ThingRegistryTest.java

```diff
@@ -3,16 +3,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package com.aws.greengrass.clientdevices.auth.iot.registry;
+package com.aws.greengrass.clientdevices.auth.iot;
 
 import com.aws.greengrass.clientdevices.auth.api.DomainEvents;
 import com.aws.greengrass.clientdevices.auth.configuration.RuntimeConfiguration;
 import com.aws.greengrass.clientdevices.auth.exception.CloudServiceInteractionException;
-import com.aws.greengrass.clientdevices.auth.iot.Certificate;
```

Comment:
Why did this change?

Reply:
Registries were moved from `iot.registry` package to `iot` where all the IoT domain classes reside. We got rid of separate `registry` package.